### PR TITLE
feat(integration) : add support for listingId and countryCode in Rockerbox

### PR DESCRIPTION
**Pull request by yashasvibajpai**
## Description of the change

> Add support for listingId and countryCode in Rockerbox

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: v0/destinations/rockerbox/transform.js [hunk 1]
```diff
@@ -29,7 +30,9 @@ const responseBuilderSimple = (message, category, destination) => {
   } else {
     payload.action = eventsHashMap[message.event.toLowerCase()];
   }
-
+  // mapping external id
+  const externalId = getDestinationExternalID(message, "rockerboxExternalId");
```
Comment (utsabc) [inline v0/destinations/rockerbox/transform.js:34]: we need to add this to canonical name

**Review by utsabc — CHANGES_REQUESTED** [review]

**Review by utsabc — DISMISSED** [review]

File: v0/destinations/rockerbox/transform.js [hunk 2]
```diff
@@ -29,7 +30,9 @@ const responseBuilderSimple = (message, category, destination) => {
...
+  payload.external_id = externalId;
```
Comment (saikumarrs) [inline v0/destinations/rockerbox/transform.js:35]: Based on the Rockerbox team's response, it seems like we don't have to send the external ID.

**Review by saikumarrs — CHANGES_REQUESTED** [review]

File: v0/destinations/rockerbox/transform.js (hunk 2 above)
Comment (yashasvibajpai) [reply v0/destinations/rockerbox/transform.js]: We have removed externalId for now, so that we can release with the other 2 supported fields.
CC @utsabc

**Review by utsabc — DISMISSED** [review]

File: v0/destinations/rockerbox/transform.js [hunk 3]
```diff
@@ -6,7 +6,8 @@ const {
   constructPayload,
   getSuccessRespEvents,
   getErrorRespEvents,
-  getHashFromArray
+  getHashFromArray,
```
Comment (krishna2020) [inline v0/destinations/rockerbox/transform.js:9]: Is `getDestinationExternalID` being used ?

**Review by krishna2020 — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]

File: v0/destinations/rockerbox/transform.js (hunk 3 above)
Comment (yashasvibajpai) [reply v0/destinations/rockerbox/transform.js]: Removing this as it is unused now.

**Review by krishna2020 — APPROVED** [review]

**Review by utsabc — APPROVED** [review]

**Pull request merged**